tests/Filter/StringFilterTest.php

```diff
@@ -195,4 +194,22 @@ public function caseSensitiveDataProvider(): array
 
         ];
     }
+
+    /**
+     * @group legacy
```

Comment:
What to do in the next major about this test?

Reply:
I added a `NEXT_MAJOR` comment.